tests/conftest.py

```diff
@@ -225,6 +225,10 @@ def wrapper(*args, **kwargs):
         try:
             timeout(1.0)(func)(*args, **kwargs)
         except Exception as e:
+            # timeout_decorator is not Windows compatible, because it relies on process signals that
```

Comment:
Good to know, thanks! I found this related issue: https://github.com/pnpnpn/timeout-decorator/issues/45
And it looks like there's a `use_signals=False` that will use multiprocessing timeouts instead of process signals. The only downside is lower time granularity, which we don't care about here.